lib/modules/versioning/ubuntu/distribution.ts

```diff
@@ -0,0 +1,42 @@
+import dataFiles from '../../../data-files.generated';
+
+export type UbuntuDistroInfo = Record<string, string>;
+
+// Data file generated with:
+// ubuntu-distro-info --all -f | sed -r 's/Ubuntu|"|LTS //g; s/([0-9]+.[0-9]+) /\1=/; s/.*/\L&/; s/(=[a-z]*) [a-z]*/\1/g; s/^[ \t]*//' | jo
+const ubuntuJsonKey = 'data/ubuntu-distro-info.json';
+
+const ubuntuDistroInfo: UbuntuDistroInfo = JSON.parse(
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  dataFiles.get(ubuntuJsonKey)!
+);
+
+const codenameToVersion = new Map<string, string>();
+const versionToCodename = new Map<string, string>();
```

Comment:
Isn't `ubunuDistroInfo` the same? We should hold unused object on module scope.

Reply:
removed that as well :)